### ISSUE TITLE
Warn about unsupported image types in the media/large_media area

### ISSUE THREAD

**Issue by permcody** [idaholab/moose#9993]
## Rationale

Enhancement to report errors for unsupported media types in MooseDocs (PDF).

## Description

If a user references certain image types such as "pdf" from a mark down file, no errors are issued from moose docs but the image isn't rendered either. This leads to confusion on the part of the developer.

## Impact

Better diagnostics to assist in writing rich documentation.

**Comment by kyriv-lab** [comment]
The robot had this to say on this issue and current code: 
"
Findings — is this an actual problem?

Partially. The premise "no errors are issued" is not quite accurate in the current code, but the underlying diagnostic problem is real:

- All media goes through the !media command (there's no ![](...) markdown image syntax). A reference like !media diagram.pdf (or .tiff, .bmp, any unregistered extension) maps to subcommand pdf, which isn't registered, so dispatch raises at command.py:130: "The following command combination is unknown: 'media pdf'."
- That error is logged (LOG.error) and does render a visible moose-exception marker on the page. So it's not silent — but the message is cryptic. "unknown command combination: media pdf" gives the developer no hint that the real issue is an unsupported media type, nor what to do about it. That matches the issue's "confusion on the part of the developer" / "better diagnostics" framing.

Least-invasive solution

Use the existing SUBCOMMAND = "*" wildcard catch-all mechanism (already used by config.py, materialicon.py, package.py, listing.py). Register one small media/* handler that fires only for unrecognized extensions and raises a clear, media-specific error listing the supported types. This covers all unrecognized formats, not just PDF. No changes to the shared dispatcher; ~15 lines in media.py.

One thing worth flagging: this raises a tokenize error (consistent with how !media already treats unknown subcommands), so it will surface as an error in the build log and as a moose-exception marker on the page — it does not make the build hard-fail by itself. If you'd instead want unsupported media to be a build-breaking error, that's a separate, more invasive change to error-handling policy. Let me know if that's the intent.
"

**Comment by kyriv-lab** [comment]
@cticenhour should we close this issue?

**Comment by cticenhour** [comment]
Yes - looks like PR verbiage didn't trigger automatically. Thank you!
